modules/oe_translation_poetry/src/Form/PoetryCheckoutFormBase.php

```diff
@@ -363,11 +370,14 @@ protected function redirectBack(FormStateInterface $form_state): void {
    * It uses the configured prefix, site ID and the title of the Job (one of the
    * jobs as they are identical).
    *
+   * @param \Drupal\oe_translation_poetry\PoetryJobQueue $queue
+   *   The job queue.
+   *
    * @return string
    *   The title.
    */
-  protected function createRequestTitle(): string {
-    $jobs = $this->queue->getAllJobs();
+  protected function createRequestTitle(PoetryJobQueue $queue): string {
```

Comment:
Could just receive the title (see comment above).

Reply:
no, but it could only receive a job to create the title. So you can pass just that.